org-journal.el

```diff
@@ -399,14 +440,15 @@ previous day's file to the current file."
                  (backward-char)
                  (setq org-map-continue-from (point))
                  subtree))))
+        (re-search-backward org-journal-created-re nil t)
```

Comment:
Does this work for daily journals, which don't have a created property?

Reply:
Sure! The error will be ignored (third argument -> noerror is set to `t'). But if you like I can wrap it inside a conditional!?

Reply:
Ok, no need to change it. Just wanted to make sure this is not a bug.